core/consensus/aura/src/lib.rs

```diff
@@ -465,7 +451,7 @@ impl<B: Block, C, E, I, P, Error, SO> SlotWorker<B> for AuraWorker<C, E, I, P, S
 /// This digest item will always return `Some` when used with `as_aura_seal`.
 //
 // FIXME #1018 needs misbehavior types
-#[forbid(deprecated)]
+#[forbid(warnings)]
```

Comment:
prefer to have these at crate roots rather than on individual items. otherwise it can get messy to keep track